components/brave_rewards/browser/publisher_info_database_unittest.cc

```diff
@@ -618,15 +619,15 @@ TEST_F(PublisherInfoDatabaseTest, GetPanelPublisher) {
    * Publisher ID is missing
    */
   ledger::ActivityInfoFilter filter_1;
-  EXPECT_EQ(publisher_info_database_->GetPanelPublisher(filter_1),
+  EXPECT_EQ(publisher_info_database_->GetPanelPublisher(filter_1.Clone()),
             static_cast<ledger::PublisherInfoPtr>(nullptr));
 
   /**
    * Empty table
    */
   ledger::ActivityInfoFilter filter_2;
   filter_2.id = "test";
-  EXPECT_EQ(publisher_info_database_->GetPanelPublisher(filter_2),
+  EXPECT_EQ(publisher_info_database_->GetPanelPublisher(filter_2.Clone()),
```

Comment:
`filter_2.Clone()` should use `std::move(filter_2)` as `filter_2` is no longer used

Reply:
fixed